core/src/main/scala/org/apache/gearpump/util/Graph.scala

```diff
@@ -17,57 +17,52 @@
  */
 
 package org.apache.gearpump.util
-
-import org.apache.gearpump.partitioner.Partitioner
-import org.apache.gearpump.util.Graph.Edge
-import org.jgrapht.Graphs
-import org.jgrapht.graph.DefaultDirectedGraph
-import org.jgrapht.traverse.TopologicalOrderIterator
-import upickle.{Reader, Js, Writer}
-import org.slf4j.Logger
-
 import scala.collection.JavaConversions._
+import scala.collection.mutable
 import scala.language.implicitConversions
 
 /**
  * Application DAG
  */
+class Graph[N, E](vertexList: List[N], edgeList: List[(N, E, N)]) extends Serializable{
```

Comment:
Will this be a candidate for scalajs? It would be valuable to use Graph operations on the client-side which would simplify some of the things that streamingdag.js (or StreamingDag.js) does.